.ci/linux-steps.yaml

```diff
@@ -49,7 +49,7 @@ steps:
   displayName: 'Build models'
 
 # Run CTests.
-- script: cd build/tests/ && sudo CTEST_OUTPUT_ON_FAILURE=1 ctest -T Test .
+- script: cd build/ && ./models_test -x 50
```

Comment:
I think it's `bin/models_test`, you can add a `cd build/ && ls && bin/models_test -x` to list the files in the directory.

Reply:
Yup yup it is that, and i did that with this commit: https://github.com/mlpack/models/pull/69/commits/ea0f8165f49375225f0c4263fdecdbc8713309e2 but again i messed this up a bit so tomorrow morning would be a better option.